The dissertation has focused on the development, implementation, and application of what is now called LT-FH++. LT-FH++ extends the previously published LT-FH method, which is itself an extension of the classical liability threshold model by Falconer. LT-FH extended the LTM such that it models family members for binary traits, which allowed for an estimate of a genetic liability that, when used as the outcome in a GWAS, provided a significant power increase. LT-FH++ extensions this framework even further by also accounting for age of onset in cases or age in controls, sex, and birth year. These things are accounted for through population representative cumulative incidence proportions that are stratified by sex and birth year. This leads to a threshold in the LTM that is unique for each individual, which has not previously been done. Since every individual has a unique threshold, a more computationally efficient sampling strategy had to be implemented than what was used in LT-FH. As a result, LT-FH++ utilises a Gibbs sampler that samples from a truncated multivariate normal distribution that allows for arbitrary thresholds. Furthermore, the implementation is parallelizable, which allows it to better utilise modern CPUs with many cores or high performance computing clusters.

In the first project, most of the code base and methodological development work was done. It culminated in the publication of the LT-FH++ method, which refined a liability that was informed by family history \textit{and} age of onset, sex, and birth year for each included individual. LT-FH++ performed between $ 4\% $ and $ 18\% $ better than LT-FH in terms of identifying the true causal associations in a simulated GWAS setup. Both LT-FH and LT-FH++ outperformed the conventional case-control status and the GWAX phenotype. For a real-world analysis, mortality in the UKBB was analysed and four psychiatric disorders from iPSYCH. For mortality, LT-FH++ significantly boosted power compared to LT-FH and case-control status. LT-FH++ was able to identify $ 10 $ genome-wide significant SNPs, while LT-FH identified $ 2 $ and case-control status identified $ 0 $. In iPSYCH, the difference between the three phenotypes was modest. There are likely several reasons for the lack of power gain over case-control status by LT-FH and LT-FH++, such as low family history prevalence and more polygenic disorders. Additional simulation studies also revealed that the mortality setup in UKBB was a near perfect scenario for LT-FH++, since it benefits from a high prevalence in either the genotyped individuals or in the family history.

The second project examined the best way to include age of onset in a GWAS setting. This meant comparing the model underlying LT-FH++, here called ADuLT as no family history was used, to other time-to-event GWAS methods. The simplest and most commonly used time-to-event GWAS method is the Cox proportional hazards model. Since the Cox proportional hazards models are computationally intensive most implementations are unable to handle more than $ 100,000 $ individuals, and we will use the most computationally efficient implementation called SPACox to represent these models. We will compare the performance of ADuLT, SPACox, and case-control status in a linear regression. We simulated genotypes and assigned phenotypes and age of onset under both the proportional hazards model and the LTM. One would expect the LTM models to perform the best when phenotypes were assigned with the LTM, and vice versa, which is also what we observed. However, when we emulated case ascertainment, meaning we downsampled the controls such that we had an equal number of cases and controls, we observed a disproportionate loss in power for SPACox. Conventionally, IPW would be used to account for the ascertainment, however it had no effect here and SPACox still performed worse than simple linear regression, even under the proportional hazards model. The same disproportionate loss of power was also observed in real-world analysis of iPSYCH disorders. As a result, we do not recommend proportional hazards to identify genome-wide significant SNPs, instead a simpler linear regression or ADuLT in a GWAS method of choice should be used.

The third and final project examined the predictive value of family history. Normally, a binary family history variable is used, such that an affirmative value is given to individuals with at least one case in their considered family, e.g.\ first degree relatives. This was compared with the PRS of the considered phenotype, as well as the LT-FH++ phenotype without any information on the index individuals. This means the LT-FH++ estimates the genetic liability solely based on the family history. We assessed the predictive value with the partial $ R^2 $ in a regression model. A base model was used with age, sex, and the first $ 20 $ PCs. Then a model with either of the family history phenotypes was considered, a model with the PRS, and a model with any combination of these. In short, LT-FH++ had an increased partial $ R^2 $ compared to the PRS model of $ 19.6\% $ across the $ 8 $ considered disorders. The binary family history variable had a $ 65.2\% $ \textit{decrease} compared to the PRS model. The model with both the binary family history variable and the LT-FH++ phenotype had almost the same predictive value as the model with only the LT-FH++ phenotype. Interestingly, the model with the LT-FH++ phenotype and the PRS had an almost additive increase in their predictive value, indicating that they capture independent genetic signals. The model with all three predictors performed nearly identically to the model with just the LT-FH++ phenotype and the PRS, indicating that all signal is captured by those predictors. 
 
As most psychiatric disorders have a high genetic correlation, we also considered a regression model that included correlated phenotypes in the prediction model. The average of these phenotypes were higher than their single trait equivalent, but difference between LT-FH++ and the binary family history variable had largely disappeared, making both variables equally predictive. The model with both family history phenotypes was had nearly the same predictive power as either of the family history models, meaning no new information was gained by using both. Combining the multi trait PRS model with either of the family history phenotypes resulted in a model with close to the sum of the PRS and family history model's predictive values. This means that even for correlated phenotypes, the genetic signal captured by the PRS and the family history phenotypes appear to be nearly independent.

In summary, we have successfully developed, implemented, and applied the LT-FH++ method in a number of different areas. The LT-FH++ method has provided improvements in each of the three applications that have been considered, while remaining computationally efficient. While family history and age of onset is not commonplace in all biobanks yet, we have demonstrated that it is a worthwhile investment, as power to detect associations in a GWAS setting, with or without family history, and the predictive value of family history have been increased by using the LT-FH++ phenotype instead of the binary variables that are commonly used.

