\begin{enumerate}
	\item what is the best way to utilise AOO information ?
	\item comment on the simulations results with and without ascertainment
	\item comment on the ipsych analysis
	\item IPW did not fix the simulation results
\end{enumerate}


The purpose of this paper was to examine the best way to include the age of onset information in a GWAS setting. The gold standard when modelling time to event is some kind of survival analysis. However the adoption of methods such as Cox proportional hazards have been limited for GWAS. One of the main limiting factors for such models is the computational cost associated with the analysis. Recent advances have allowed for Cox proportional hazards models and frailty models to be used on UKBB-sized biobanks \cite{bi2020fast,dey2022efficient}. Both methods utilise a saddle point approximation \cite{daniels1954saddlepoint}, as it provides a computationally efficient way to calculate p values. The implementation of the proportional hazards model proposed by Bi et al. is called SPACox and is available as an R package. The frailty model proposed by Dey et al. is called GATE and has been implemented in R and Rcpp, which is a R-wrapper around C++. Bi et al. show previous implementations take more than $ 300 $ CPU hours for an analysis of $ 400,000 $ individuals and $ 20 $ million SNPs, which has been reduced to just $ 30 $ hours with SPACox.

Since a proportional hazards model and a liability threshold model are fundamentally different, we did not want to unfairly favour one method over the other. Therefore, we performed simulations under both generative models, meaning genotypes were simulated in the same way, but two separate analysis were run where the phenotype had been assigned with different generative models. One would expect that the LTM based methods would perform the best under the LTM model, and vice versa, which is also what we experienced. Interestingly, we found that SPACox was disproportionately affected by case ascertainment, suffering far more than the LTM based methods. SPAcox had the lowest power under both generative models and all prevalences considered except for the least ascertained parameter setup under the proportional hazards model. Conventionally, inverse probability weighing would be used to account for any form of ascertainment, however, it did not increase power. In fact, IPW did not seem to change the power in any noticeable way compared to SPACox. The SPACox method does not support IPW, which means the IPW simulations were performed on all causal SNPS and fewer null SNPs, and with the survival\cite{Therneau2020-xf} package's \texttt{coxph} function instead.

The test statistics used with IPW is based on a Wald test\cite{survivalVignette}, which means the test statistic is the estimate divided by the standard error. When performing IPW, the estimate will remain unbiased, but estimating the standard error can be difficult \cite{austin2016variance}. \textbf{THE VARIANCE ESTIMATE IS BASED ON HORVITZ-THOMPSEN}


%The cox ph test is a wald test, i.e. effect size / stderr. The effect size is estimated unbiased by weighing samples. the std err is based on a horvitx-thompsen estimator (https://cran.r-project.org/web/packages/survival/vignettes/survival.pdf) p. 39 , chap 2.7. 
% $ \hat{V}ar(\hat{\tau}_\pi)=\sum\limits_{i=1}^v \left( \dfrac{1-\pi_i}{\pi^2_i} \right) y^2_i + \sum\limits_{i=1}^v \sum\limits_{j\neq i} \left( \dfrac{\pi_{ij}-\pi_i \pi_j}{\pi_i \pi_j}\right)\dfrac{1}{\pi_{ij}} y_i y_j $


