The use of family history as a predictor of disease risk has long been a subject of interest in epidemiology and preventive medicine. While family history captures both environmental and genetic variation, including the so-called "missing heritability", its predictive power is often limited by the use of binary indicators to account for the presence or absence of a particular disorder in the family. 

In this study, we repurposed to utilise the LT-FH++ model to quantify individual family history risk and estimate individual family liabilities (FL). This means estimating the liability of the index person, but ignoring the disease status in the individual, such that all information is derived from the family members. It is similar to the previously proposed family genetic risk scores by Kendler et al.\cite{kendler2021family} and the FL estimated under the LT-FH++ model is a time-to-event model that accounts for differences in prevalences by birth year and sex, as well as accounting for age. By using a model-based approach, we aim to improve the interpretability and predictive power of family history as a risk factor, as the Kendler et al. is a heuristic approach. We evaluate the performance of our method using data from the Danish registers and iPSYCH study, and compare it to binary family history indicators, as well as PRS. Our results show that FL estimates have improved predictive accuracy over standard binary family history indicators. We note that this result is in stark contrast to previous results by Hujoel et al.\cite{hujoel2022incorporating}, which found that estimating family risk using a multivariate liability threshold model provided little or no benefit over binary family history indicators. However, we believe this is due to both more detailed family history information available in the Danish registers (parents, siblings, children, paternal and maternal half-siblings and grandparents), as well as our proposed model accounts for sex, birth year, and age or age-of-onset for all family members. 

We further proposed combining FLs for multiple correlated health outcomes to improve their predictive accuracy. We found this approach to provide less benefit over the comparable approach of combining family history indicators for multiple correlated health outcomes. While the predictive accuracy of the extension to genetically correlated health outcomes was not improved, it still has potential as an accurate liability outcome in a GWAS. As the multi-trait extension estimates a single value, this application is of particular interest and an area of future research.
Similar to previous work\cite{mars2022systematic,wolford2021utility,hujoel2022incorporating}, we found that PRS and family risk measures captured largely independent information. We note that there are several reasons for this independence between PRS and FL. First, the accuracy of the PRS is limited by heritability explained by the genotyped variants, whereas FL can (in theory) capture any additive genetic variance (full heritability) as well as shared environmental effects. Second, FL and PRS are trained on different data, with PRS using external summary statistics and genotypes and FL using register information. Third, given current sample sizes, their absolute variance explained is small which makes it unlikely that they capture the \enquote{same} variance. 

There are several limitations to our study, some of which we aim to address in future or ongoing research. First, both the Binary family history and the FL variables are subject to limitations on available family history in biobanks or registers. If no or only limited family history information is available, these methods are unlikely to provide a significant increase in prediction accuracy. For example, UKBB only has family history available for $ 12 $ phenotypes, with full family history information only available for parents. Second, the predictive accuracy of FL is unlikely to have unbounded potential for improvement as more family information becomes available, as families are rarely very large and are not likely to increase substantially in the future. However, our work suggests that combining family history for multiple outcomes may further improve FL. Third, the model underlying FL assumes that the full additive (narrow sense) heritabilities and genetic correlations are known. However, these may not always be available, nor be easily estimated in the family data. We aim to address this limitation by estimating these parameters directly from the family data. Finally, using the average PC value as a reference when calculating the genetic distances used to stratify prediction accuracies could be a poor reference choice, as it may not match individuals of Danish genetic ancestry well. We aim to remedy this by using a more precise Danish genetic ancestry reference using a similar approach as Priv√© et al.\cite{prive2022portability}.

While family history is still not widely available in biobanks, an ever-increasing number of biobanks have some level of family history information, and this trend is likely to continue. The biobanks that already have family history information may continue to expand on them, further increasing their utility. As illustrated by the multi-trait analysis performed here, utilising correlated phenotypes, either through family history or PRS, has a significant potential to improve overall prediction of a particular phenotype. Combining FLs and PRS has the potential to increase prediction even further, as they conceptually estimate the same thing, while being largely independent.
