The first paper proposed the method LT-FH++, which is an extension of the previously proposed LT-FH method by Hujoel et al\cite{hujoel2020liability}. The most notable difference between LT-FH and LT-FH++ is the ability to account for age-of-onset for cases or age for controls, sex, and birth year, as well as the same information in the included family members. The LT-FH method does not consider sex or age in parents, meaning they have the same thresholds. It also uses the same threshold for the index person and siblings and does not distinguish on age or sex differences. LT-FH++ is also able to account for siblings individually rather than considering the number of siblings and an \enquote{at least one affected sibling} indicator. This way of coding siblings in LT-FH is likely due to the way sibling information is coded in the UKBB, which was the main application of the LT-FH paper. Considerable changes have also been made to the sampling strategy to allow for the increased flexibility in the family and the use of personalised thresholds to be scalable to millions of individuals.  The changes LT-FH++ proposed increased the number of unique configurations considerably, as each individual now has a unique set of family members and thresholds. The sampling strategy used for LT-FH would be computationally intractable for LT-FH++, since LT-FH only needed to estimate a liability for a handful of configurations.


\subsection{Simulation Results}
We performed simulations to assess the power and false discovery rate of LT-FH++ against LT-FH and a case-control status to detect causal SNPs in a linear regression GWAS. The simulations are based on simulated genotypes, where we simulated a pair of parents and one offspring with no siblings. We used parameters similar to the ones used in the LT-FH paper to ensure compatibility between findings. The simulated genotypes had a heritability on the liability scale of $ h^2 = 0.5 $, a population prevalence of $ 5\% $. Unlike in the LT-FH paper, we used a higher prevalence in one of the simulated sexes, but the combined prevalence would still be $ 5\% $. The case ratio was $ 1{:}4 $ between sexes, and it was also present in the parents. We also considered a population prevalence of $ 10\% $, but those results are not shown here. The genotypes consisted of $ 100,000 $ individuals, each with $ 100,000 $ independent SNPs where $ 1000 $ SNPs were causal, i.e.\ they had a simulated effect size different from $ 0 $. The simulation results shown in \cref{fig:LTFHppSimulationResults} are based on $ 10 $ replications of each simulation scenario. Case ascertainment is common in biobanks, which means there is a higher (or lower) prevalence of a phenotype of interest in the biobank compared to the rest of the population (See \cref{sec:bias} for details). We emulated case ascertainment in the simulations by downsampling the entire population until we had a subpopulation with $ 10,000 $ individuals with the same number of cases and controls.

\begin{figure}[h]
	\includegraphics[width=\textwidth]{results/boxplot_05prev_both.png}
	\caption[Simulation results for a $ 5\% $ prevalence, with and without downsampling of controls]{
		\sl A slightly modified version of Figure 2 from Paper $ 1 $\cite{pedersen2022accounting} (also \cref{app:ltfhpp}, Figure $ 2 $). Linear regression was used to perform the GWAS for LT-FH and LT-FH++, while a 1-df chi-squared test was used for case-control status. We assessed the power of each method by considering the fraction of causal SNPs with a p-value below $ 5 \times 10^{-8} $. Here, GWAS refers to case-control status and LT-FH and LT-FH++ are both without siblings. Downsampling refers to downsampling the controls such that we have the same number of cases and controls, i.e.\, we have $ 10,000 $ individuals in total for a $ 5\% $ prevalence and $ 20,000 $ individuals for a $ 10\% $ prevalence.}
	\label{fig:LTFHppSimulationResults}
\end{figure}

The simulations show a modest power increase in favour of LT-FH++ over LT-FH in the full sample, with an average power increase across the $ 10 $ simulations of $ 4\% $. Both LT-FH and LT-FH++ has an average power increase of more than $ 50\% $ compared to the case-control status used in \textit{GWAS}, making either method vastly better. However, case ascertainment has a significant impact on the power ratio between LT-FH and LT-FH++. When case ascertainment is present in a biobank, the average power increase of LT-FH++ over LT-FH increased to $ 18\% $.

\newpage

\subsection{Real-World Analysis}
\begin{wrapfigure}{R}{10cm}
	%	\includegraphics[width=0.7\textwidth]{results/manhattanPlot_mortality.pdf} % adds a lot of loading 
	\includegraphics[width=10cm]{results/manhattanPlot_mortality.png}
	\caption[Manhattan plots for LT-FH++, LT-FH, and case-control GWAS of mortality	in the UK Biobank]{
		\sl The Manhattan plots display a Bonferroni corrected significance level of $ 5\times 10^{-8} $ and a suggestive threshold of $ 5\times 10^{-6} $. The genome-wide significant SNPs are coloured in red. The diamonds correspond to top	SNPs in a window of size $ 300,000 $ base pairs. The plot is originally Figure $ 3 $ from Paper $ 1 $\cite{pedersen2022accounting} (also \cref{app:ltfhpp}, Figure $ 3 $).}
	\label{fig:LTFH++_manhattanMortality}
\end{wrapfigure}

LT-FH++ was also applied to four of the main psychiatric disorders in iPSYCH and to mortality in UKBB. The mortality GWAS in UKBB resulted in $ 0 $ genome-wide significant SNP for simple linear regression, $ 2 $ for LT-FH, and $ 10 $ for LT-FH++. The Manhattan plot for mortality can be found in \cref{fig:LTFH++_manhattanMortality}.

The GWAS in iPSYCH did not provide nearly as large of an increase in power for LT-FH++ or LT-FH over simple linear regression. In fact, we did not see any notable improvement over simple linear regression of the case-control status. The Manhattan plot for ADHD in iPSYCH can be found in \cref{fig:LTFH++_manhattanADHD}. We did find $ 7 $ genome-wide significant SNPs for ADHD using LT-FH++ and $ 5 $ for LT-FH and case-control status, but the two additional associations for LT-FH++ were very close to genome-wide significance for the other two outcomes as well.

Through additional simulations we found that one can expect the most \textit{relative} power gain with LT-FH++ over LT-FH if the in-sample prevalence is high in either family members or the index persons. This is because LT-FH++ is best able to utilise information for cases, since the CIPs provide a very accurate estimate for the full liability of an individual.